agent/bls.mjs

```diff
@@ -0,0 +1,2976 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+/**
+ * @license
+ * SKALE IMA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @file bls.mjs
+ * @copyright SKALE Labs 2019-Present
+ */
+
+import * as fs from "fs";
+import * as cc from "../npms/skale-cc/cc.mjs";
+import * as log from "../npms/skale-log/log.mjs";
+import * as owaspUtils from "../npms/skale-owasp/owasp-utils.mjs";
+import * as child_process from "child_process";
+import * as rpcCall from "./rpc-call.mjs";
+import * as shell_mod from "shelljs";
+import * as imaUtils from "./utils.mjs";
+import * as sha3_mod from "sha3";
+import * as skale_observer from "../npms/skale-observer/observer.mjs";
+
+import * as state from "./state.mjs";
+import { randomCallID } from "../npms/skale-cool-socket/socket_utils.mjs";
+
+const shell = shell_mod.default;
+
+const Keccak = sha3_mod.Keccak;
+
+const sleep =
+    ( milliseconds ) => { return new Promise( resolve => setTimeout( resolve, milliseconds ) ); };
+
+const g_secondsMessageVerifySendTimeout = 2 * 60;
```

Comment:
https://github.com/skalenetwork/internal-support/blob/main/docs/wiki/code-guidelines.md#coherent-style

Reply:
Fixed, but `g_` prefixes are left for globals